src/gui/guiFormSpecMenu.cpp

```diff
@@ -395,7 +394,35 @@ void GUIFormSpecMenu::parseList(parserData* data, const std::string &element)
 
 		if(!data->explicit_size)
 			warningstream<<"invalid use of list without a size[] element"<<std::endl;
-		m_inventorylists.emplace_back(loc, listname, pos, geom, start_i, data->real_coordinates);
+
+		FieldSpec spec(
+			"",
+			L"",
+			L"",
+			258 + m_fields.size()
```

Comment:
Couldn't these be placed in one line?

Reply:
They could, but all field specs are created in this format. Hence changing one would require changing all, making this PR even larger.

Reply:
Understood; I noticed that too.